.gitignore

```diff
@@ -1,15 +1,3 @@
 .DS_Store
```

Comment:
Add lib/ and pretty much everything application from here: https://github.com/github/gitignore/blob/master/Python.gitignore